### DISCUSSION TITLE
Surrealdb (surrealdb.com) drivers/connectors

### DISCUSSION THREAD

**Discussion by forensic3000** [dbeaver/dbeaver#36533] [General]
Are there drivers / instructions  for connecting to surrealdb  (surrealdb.com)?

**Comment by serge-rider** [comment]
Surrealdb doesn't have a JDBC or ODBC driver so it is not that easy. 
However you could create a feature request. We will add it as soon as the vendor will release xDBC driver or we will develop one by ourselves (as we did with libsql driver).
